pkg/skaffold/docker/parse.go

```diff
@@ -47,6 +52,33 @@ const (
 // For testing.
 var RetrieveConfig = retrieveImageConfig
 
+type DockerfileDepResolver struct{}
+
+var DefaultDockerfileDepResolver = &DockerfileDepResolver{}
```

Comment:
You can skip the empty struct and alias the function itself.

var DefaultDepResolver = GetDependencies

and avoid the struct method.